faunadb-java-dsl/src/main/java/com/faunadb/client/types/Value.java

```diff
@@ -348,7 +348,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-      return format("ObjectV(%s)", values);
+        return values.entrySet()
+            .stream()
+            .map(e -> String.join(" = ", Arrays.asList(e.getKey(), e.getValue().toString())))
```

Comment:
while `values` can't be null (it's checked in constructor), its components seem like they can be, so I'd rather not try to invoke `toString()` on `e.getValue()` here in case it's null. I'd use `String.valueOf(e.getValue())` which'll render null as "null".

Reply:
Interesting point... if they _are_ null, we'd probably like them to be `NullV`, which renders as "null" anyhow. So, I don't see a problem with that reasoning. :)